### PR TITLE
make the example running well on AWS

### PR DESCRIPTION
**Pull request by Yancey0623**

### CODE REVIEW

File: doc/install.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+- MacOS
+
+```bash
+wget -O kubectl https://storage.googleapis.com/kubernetes-release/release/v1.9.0/bin/darwin/amd64/kubectl
+chmod +x ./kubectl
+sudo mv ./kubectl /usr/local/bin/kubectl
+```
+
+- Linux
```
Comment (typhoonzero) [inline doc/install.md:20]: How about telling readers that if they are using windows, use a docker environment to achieve this?

File: doc/install.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+``` bash
+kubectl version
+```
+
+### AWS
+
+If you are trying to deploy EDL on a real Kubernetes cluster, [kops](https://github.com/kubernetes/kops) is
+an easy way to get a production grad Kubernetes.
+
+You can checkout the [tutorial](https://github.com/kubernetes/kops/blob/master/docs/aws.md#configure-dns) for the details of lunching a Kubernetes cluster on AWS.
```
Comment (typhoonzero) [inline doc/install.md]: launching

File: doc/install.md (hunk 1 above)
Comment (Yancey0623) [reply doc/install.md:20]: The user can achieve a binary file adapted windows, will add the section.

File: doc/install.md (hunk 2 above)
Comment (Yancey0623) [reply doc/install.md]: Done.

File: doc/install.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+### AWS
+
+If you are trying to deploy EDL on a real Kubernetes cluster, [kops](https://github.com/kubernetes/kops) is
+an easy way to get a production grad Kubernetes on AWS.
+
+You can checkout the [tutorial](https://github.com/kubernetes/kops/blob/master/docs/aws.md) for the details of launching a Kubernetes cluster on AWS.
+
+- Prepare kops environment
+
+- Configurate DNS
```
Comment (typhoonzero) [inline doc/install.md:61]: Is this nessesary or optional?

File: example/examplejob.yaml [hunk 4]
```diff
@@ -3,17 +3,17 @@ kind: TrainingJob
 metadata:
   name: example
 spec:
-  image: "paddlepaddle/paddlecloud-job"
+  image: "yancey1989/edl-example"
```
Comment (typhoonzero) [inline example/examplejob.yaml]: image name?

File: example/train_ft.py [hunk 5]
```diff
@@ -77,7 +77,7 @@ def convolutional_neural_network(img):
 def main():
     etcd_ip = os.getenv("ETCD_IP")
     etcd_endpoint = "http://" + etcd_ip + ":" + "2379"
-    paddle.init()
+    paddle.init(trainer_count=1)
```
Comment (typhoonzero) [inline example/train_ft.py]: this use only one core.

**Review by typhoonzero — COMMENTED** [review]
LGTM, I can help test this out after it's merged.

File: doc/install.md (hunk 3 above)
Comment (Yancey0623) [reply doc/install.md:61]: It's a necessary and important one.

File: example/examplejob.yaml (hunk 4 above)
Comment (Yancey0623) [reply example/examplejob.yaml]: Done, changed to the official image name.

File: example/train_ft.py (hunk 5 above)
Comment (Yancey0623) [reply example/train_ft.py]: Done.

**Review by typhoonzero — APPROVED** [review]

**Pull request merged**